hazelcast-sql-core/src/test/java/com/hazelcast/sql/impl/expression/math/DoubleBiFunctionIntegrationTest.java

```diff
@@ -55,7 +56,8 @@
     @Parameterized.Parameters(name = "mode: {0}")
```

Comment:
```suggestion
    @Parameterized.Parameters(name = "function: {0}")
```

Reply:
I would not like to change the style given in `DoubleFunctionIntegrationTest`